docs/environment-variables.md

```diff
@@ -155,7 +155,14 @@ Kubo tries to reuse the same source port for all connections to improve NAT
 traversal. If this is an issue, you can disable it by setting
 `LIBP2P_TCP_REUSEPORT` to false.
 
-Default: true
+Default: `true`
+
+## `LIBP2P_TCP_MUX`
```

Comment:
Why does this exist? It seems like in theory this could just be turned off if there weren't 2 TCP listeners sharing a port. Is the issue that plumbing and parsing of the addresses via the current fx setup is too painful and we don't want to hold back shipping this?

Reply:
This is to confirm error is caused by `libp2p.ShareTCPListener()` – see https://github.com/ipfs/kubo/pull/10565#discussion_r1893232469